ffmpeg/lpms_ffmpeg.c

```diff
@@ -11,6 +11,7 @@
 
 // Not great to appropriate internal API like this...
 const int lpms_ERR_INPUT_PIXFMT = FFERRTAG('I','N','P','X');
+const int lpms_ERR_INPUT_CODEC = FFERRTAG('I','N','P','C');
```

Comment:
Usually for custom errors we need to add an error string for golang as well, see:

https://github.com/livepeer/lpms/blob/02ac425f2b6563a67189929a049dc489dbd936a9/ffmpeg/ffmpeg_errors.go#L31-L40